test/Microsoft.AspNetCore.SignalR.Common.Tests/Internal/Protocol/HubMessageHelpers.cs

```diff
@@ -11,6 +11,11 @@ internal static class HubMessageHelpers
         // This lets you add headers to a hub message and return it, in a single expression.
         public static HubMessage AddHeaders(IDictionary<string, string> headers, HubInvocationMessage hubMessage)
         {
+            if (hubMessage.Headers == null)
+            {
+                hubMessage.Headers = new Dictionary<string, string>();
```

Comment:
Skip if there are no values in `headers`?

Reply:
This is a test 😄